index.html

```diff
@@ -51,5 +52,183 @@ <h3>LET’S CONNECT</h3>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonica1.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+      <div class="grid-item">
+        <div class="image-wrapper">
+          <img src="/images/tonic.png" alt="work-image" />
+        </div>
+
+        <h1>Tonic</h1>
+        <div class="work-details">
+          <p class="first-detail">CANOPY</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">Back End Dev</p>
+          <div class="spacer margin-spacer"></div>
+          <p class="margin-spacer">2015</p>
+        </div>
+        <h2>
+          A daily selection of privately personalized reads; no accounts or
+          sign-ups required.
+        </h2>
+        <ul>
+          <li>HTML</li>
+          <li class="margin-spacer">CSS</li>
+          <li class="margin-spacer">JavaScript</li>
+        </ul>
+        <button class="margin-spacer">see Project</button>
+      </div>
+    </section>
+    <section class="about-myself">
+      <h1>About Myself</h1>
```

Comment:
kindly use an `h2` or `h3` tag for the headings of each of your projects here,as `h1` tag is only to be used for the first or main title in a webpage and for semantic reasons the hierarchy of header tags is from `h1-h6` in that order, since you have an `h2` tag above you cannot have another `h1` tag here 👍